embeds/.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+# https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html
+Name=Anywherelan {VERSION}
```

Comment:
I am not sure we need version in the name, but it's definitely would be nice to have somewhere in meta (Comment probably?)